src/test/kotlin/org/jetbrains/research/transformations/SetupParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.transformations
+
+import java.io.File
+import java.util.logging.Logger
+
+object SetupParser {
+    val LOG = Logger.getLogger(this.javaClass.name)
```

Comment:
You can omit using `this`  together with `javaClass` as you did on line 15 while getting a resource. Also please, make LOG a **private** variable